### PR TITLE
Add user

### PR DESCRIPTION
**Pull request by ernstBonn**
add user register & login
remove image upload 'feature'

sry for this awkward PR. In the future i'll keep my PRs shorter

### CODE REVIEW

**Comment by ernstBonn** [comment]
PR for login & register
sry for this back-and-forth committing; ->SonarCloud 
In the future i'll keep my PRs smaller

File: backend/src/main/java/com/example/backend/controller/UserController.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+@RestController
+@RequestMapping("/api/")
+@RequiredArgsConstructor
+public class UserController {
+
+    private final UserService userService;
+
+    //    @PostMapping("register")
+//    public void addUser(@RequestBody AppUserDTO user) {
+//    }
```
Comment (bremerjung) [inline backend/src/main/java/com/example/backend/controller/UserController.java:19]: unused code

File: backend/src/main/java/com/example/backend/model/AppUserDTO.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+
+public class AppUserDTO {
+
+    private String id;
+    private String username;
+//    private String password;
```
Comment (bremerjung) [inline backend/src/main/java/com/example/backend/model/AppUserDTO.java:19]: unused code

File: backend/src/main/java/com/example/backend/service/UserService.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+//            throw new IllegalArgumentException("Username already taken");
+//        }
+//        PasswordEncoder encoder = Argon2PasswordEncoder.defaultsForSpringSecurity_v5_8();
+//        AppUser realUser = new AppUser(
+//                setUUID.setUUID(),
+//                user.getUsername(),
+//                encoder.encode(user.getPassword()),
+//                List.of());
+//        userRepo.save(realUser);
+//    }
```
Comment (bremerjung) [inline backend/src/main/java/com/example/backend/service/UserService.java:42]: unused code

File: frontend/src/App.tsx [hunk 4]
```diff
@@ -1,34 +1,43 @@
...
-          </div>
-      </>
-  );
+        <div>
+            {/*<div className="header">*/}
+            {/*    <h1>CRATEGO</h1>*/}
+            {/*</div>*/}
+            {/*<div className="main">*/}
+            {/*    <StorageGallery storages={storages}/>*/}
+            {/*</div>*/}
```
Comment (bremerjung) [inline frontend/src/App.tsx:33]: unused code

**Review by bremerjung — APPROVED** [review]
At some places, this pull request contains commented code that is not being used. Please remove it before merging. Otherwise, good job.

**Pull request merged**